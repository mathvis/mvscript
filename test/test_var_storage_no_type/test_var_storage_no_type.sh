#!/bin/bash
helix test_var_storage_no_type.mvs test_var_storage_no_type.out --vsplit
