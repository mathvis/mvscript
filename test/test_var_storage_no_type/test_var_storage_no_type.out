(Decl (Variable (VarIdentifier "a") Nothing Nothing),ParserState {config = Configuration {debug = True, collapseOperations = True, collapseControlFlow = True, colors = True}, st = fromList [("a",VariableData {variableType = Nothing, inScope = True, isInitialized = False, isConstant = False})], fst = fromList [], context = Nothing})
